plugins/communicators/winrm/command_filters/rm.rb

```diff
@@ -31,7 +31,7 @@ def filter(command)
           if recurse
             ret_cmd = "rm #{dir} -recurse -force"
           else
-            ret_cmd = "rm #{dir} -force"
+            ret_cmd = "if (Test-Path #{dir}) {Remove-Item #{dir} -force}"
```

Comment:
Does this mean that the logic branch above this also needs to test if `dir` exists before removing it?